doc/contribute/guidelines.rst

```diff
@@ -418,9 +418,12 @@ project's style and naming conventions.
 .. _Linux kernel coding style:
    https://kernel.org/doc/html/latest/process/coding-style.html
 
-In general, follow the `Linux kernel coding style`_, with the
-following exceptions:
+In general, follow the `Linux kernel coding style`_, with the following
+exceptions:
 
+* The line length for code is 100 columns or fewer.
+* The line length for documentation is 80 columns or fewer. Longer lines for URL
+  references are an allowed exception.
```

Comment:
Just wondering if there is some reasons to limit documentation lines to 80 if we already allow 100 columns for code?

Reply:
I am also wondering what is the rationale for setting the line length limit of 80 for docs only.

Reply:
80 columns is the currently documented requirement in the [Documentation Guidelines](https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#tabs-spaces-and-indenting)

 > Keep the line length for documentation less than 80 characters to make it easier for reviewing in GitHub. Long lines because of URL references are an allowed exception.

Reply:
I believe that was left at 80 due to an oversight. The docs should have the same line length limit of 100 as code.

Reply:
Updated the Doc guidelines.